test/e2e/secret_test.go

```diff
@@ -200,27 +200,27 @@ var _ = Describe("Podman secret", func() {
 		list := podmanTest.Podman([]string{"secret", "ls", "--filter", fmt.Sprintf("name=%s", secret1)})
 		list.WaitWithDefaultTimeout()
 		Expect(list).Should(Exit(0))
-		Expect(list.OutputToStringArray()).To(HaveLen(2), ContainSubstring(secret1))
+		Expect(list.OutputToStringArray()).To(HaveLen(2), "", ContainSubstring(secret1))
```

Comment:
What do these changes do??

From a quick look it seems that the original test was incorrect (should https://github.com/cdoern/podman/blob/0dfaebf9921c32c9181db992d7221b0c8827351d/vendor/github.com/onsi/gomega/internal/assertion.go#L60 complain?) but this just shuts up the validator without actually making `ContainSubstring` effective.

Reply:
Nonononononononononono! You cannot do this. Please see the treadmill PR #13808 for actual correct proper fixes. TL;DR Expect() does not take a list, it takes a single value (and optional string description). You have to split this into multiple Expects.